### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by nitrag**
Create the ability to run in docker for faster development.
Migrate to postgresql.

### CODE REVIEW

File: boltstream/settings.py [hunk 1]
```diff
@@ -43,8 +43,7 @@
 DEBUG = ENV.bool("DEBUG", True)
 
 ALLOWED_HOSTS = ["*"]
-APP_HOST = ENV.str("APP_HOST", "localhost")
-
+APP_HOST = ENV.str("APP_HOST", "0.0.0.0")
```
Comment (eguven) [inline boltstream/settings.py]: This is an unsafe default IMO.

File: docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      context: .
+      dockerfile: Dockerfile-nginx
+    restart: unless-stopped
+    env_file: '.env'
+    ports:
+      - "8080:8080"
+    volumes:
+      - ./docker/nginx/offline.mp4:/etc/nginx/offline.mp4
+      - ./docker/nginx/default_preview.mp4:/etc/nginx/default_preview.mp4
+      - ./docker/nginx/rtmp-exec.py:/etc/nginx/rtmp-exec.py
```
Comment (eguven) [inline docker-compose.yaml]: I think maybe this wasn't included in the commit (along with other mp4 files), or is the user expected to provide them?

File: boltstream/settings.py [hunk 3]
```diff
@@ -107,9 +106,14 @@
 # https://docs.djangoproject.com/en/2.0/ref/settings/#databases
 
 DATABASES = {
```
Comment (benwilber) [inline boltstream/settings.py]: I think we should try to be database-agnostic from Django's perspective, if we can.  We should probably leave this just as the default `ENV.db_url` and then export an environment from Docker like:

```
DATABASE_URL=postgres://boltstream:boltstream@postgres:5432/boltstream
```

This way people can still just run the Django app on the local computer with sqlite for dev if they want to.  Thanks a lot for this!  Really looking great!

File: docker-compose.yaml (hunk 2 above)
Comment (nitrag) [reply docker-compose.yaml]: I'm thinking these common files should not be part of ansible directory structure. They are part of the application and should be packaged within the dockerfile. Perhaps a directory called `common` at the top-level?

@benwilber Can Ansible reference/include files outside of its own directory structure?

File: README.md [hunk 4]
```diff
@@ -57,14 +58,27 @@ You will also need to modify some variables in `terraform/terraform.tfvars`, and
...
 At the end of the Terraform deployment (might take 10-15 minutes), you will have a full self-hosted live video streaming platform with your own RTMP ingest and playback endpoints.
 
 Happy Streaming!
 
 ## Help Wanted!
 
-I put this stack together about a year ago and haven't been able to push much further on it.
-
-Ideally it could be deployed in Docker (I don't know anything about Docker or Kubernetes).  The nginx and Django stuff seems like it could be pretty easy to containerize.
-
```
Comment (kn0ll) [inline README.md:69]: 😄  👍

**Pull request closed without merging**